packages/frontend/src/components/DashboardFilter/FilterDashboardTo.tsx

```diff
@@ -24,7 +24,8 @@ export const FilterDashboardTo: FC<Props> = ({ filters, onAddFilter }) => (
                 icon={<MantineIcon icon={IconFilter} />}
                 onClick={() => onAddFilter(filter, true)}
             >
-                {friendlyName(filter.target.fieldId)} is{' '}
+                {friendlyName(filter.target.tableName)} -{' '}
+                {friendlyName(filter.target.fieldName)} is{' '}
```

Comment:
will this work for existing filters that are stored in the database but don't have the `target.fieldName` property?

Reply:
Good catch! I believe this is only working for filters without `fieldName` because of the `any` in `lightdash/packages/backend/src/controllers/savedChartController.ts:77` and the fact that we don't use it to actually get the results in the backend.

I can try and instead of adding `fieldName` to the filters type, extract it from the `fieldId`. What do you think?

Reply:
@owlas & @filp I changed this type a bit and made `fieldName` a nullable property, this way there's no need for any change in the backend

also, since the filters of `FilterDashboardTo` component come from `createDashboardFilterRuleFromField` frontend function which depends on field, `fieldName` will always exist